src/Orbit/Graphics/Context/RenderContext.cpp

```diff
@@ -22,6 +22,7 @@
 
 #include "Orbit/Core/IO/Log.h"
 #include "Orbit/Core/Platform/Android/AndroidApp.h"
+#include "Orbit/Core/Platform/Android/android_native_app_glue.h"
```

Comment:
The provided app glue has no platform guards so this will fail to compile under any other platform other than Android